ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/133__embedded_operator.robot

```diff
@@ -0,0 +1,153 @@
+*** Settings ***
+Library    String
+Library    OpenShiftLibrary
+Library    OperatingSystem
+Library    ../../../../../libs/Helpers.py
+Resource   ../../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource   ../../../../../tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot
+Resource   ../../../../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+Suite Teardown    Uninstall RHODS V2 Embedded
+
+
+***Variables***
+${cluster_type}              selfmanaged
+${image_url}                 ${EMPTY}
+${dsci_name}                 default
+${dsc_name}                  default
+
+
+*** Test Cases ***
+Install "Embedded Operator" RHODS
+    [Documentation]
+    [Tags]
+    Set Global Variable    ${UPDATE_CHANNEL}    embedded
+    Install Embedded RHODS    ${cluster_type}     ${image_url}
+
+Verify Embedded RHODS Installation
+    [Documentation]
+    [Tags]
+    RHODS Embedded Verification
+
+
+*** Keywords ***
+Install Embedded RHODS
+    [Arguments]    ${cluster_type}     ${image_url}
+    ${is_operator_installed} =  Is RHODS Installed
+    IF    ${is_operator_installed}
+        Fail    Use a clean cluster to run this installation
+    END
+    IF    "${cluster_type}" == "selfmanaged"
+        ${file_path} =    Set Variable    ods_ci/tests/Resources/Files/operatorV2/
+        ${rc}  ${out} =    Run And Return Rc And Output    oc create ns ${OPERATOR_NAMESPACE}
+        IF    ${rc}!=0    Fail
+        Log    ${out}    console=yes
+        Copy File    source=${file_path}operatorgroup_template.yaml    destination=${file_path}operatorgroup_apply.yaml
+        Run    sed -i 's/<OPERATOR_NAMESPACE>/${OPERATOR_NAMESPACE}/' ${file_path}operatorgroup_apply.yaml
+        ${rc}  ${out} =    Run And Return Rc And Output   oc apply -f ${file_path}operatorgroup_apply.yaml
+        IF    ${rc}!=0    Fail
+        Log    ${out}    console=yes
+        Remove File    ${file_path}operatorgroup_apply.yaml
+        ${image_url_bool} =    Evaluate    '${image_url}' == ''
+        IF  ${image_url_bool}
+            # Prod 2.1 build
+            Log    Installing prod 2.1 build    console=yes
+            Copy File    source=${file_path}subscription_template_21.yaml    destination=${file_path}subscription_apply.yaml  # robocop: disable
+            Run    sed -i 's/<OPERATOR_NAMESPACE>/${OPERATOR_NAMESPACE}/' ${file_path}subscription_apply.yaml
```

Comment:
there is a kw that creates the subscription and the operator group (`Install ISV Operator From OperatorHub Via CLI`) That could be extended to add the config part needed for the single namespace parameter.